config/locales/en.account_components.yml

```diff
@@ -0,0 +1,16 @@
+# Labels for account components. These are often overridden/added to in the schools
+# to match their chart string fields (e.g. Fund, Org, Department, etc.)
```

Comment:
Seems weird this is in its own super specific yaml file--what's the rationale there?

Reply:
Over time, I've been trying to find a better organization for the locales, and a lot of that is shortening up the base `en.yml`. One of the things I wanted to do was consolidate these two pieces since they're very related. There might be a better place, but this was the best I came up with.